bioconda_utils/lint/check_deprecation.py

```diff
@@ -56,3 +56,22 @@ def fix(self, _message, _data):
         self.recipe.replace('numpy x.x', 'numpy',
                             within=('requirements', 'outputs'))
         return True
+
+
+class uses_matplotlib(LintCheck):
+    """The recipe uses ``matplotlib``, but ``matplotlib-base`` is recommended
+
+    The ``matplotlib`` dependency should be replaced with ``matplotlib-base``
+    unless the package explicitly needs the PyQt interactive plotting backend.
```

Comment:
We might have to shorten this to fit in one line, depending on how the bot might render it.
But we can do this later, no need to hold off on merging this PR.